apps/web/src/components/sidebar/SidebarFooter/index.tsx

```diff
@@ -1,46 +1,21 @@
 import type { ReactElement } from 'react'
-import { useEffect } from 'react'
 
 import {
   SidebarList,
   SidebarListItemButton,
   SidebarListItemIcon,
   SidebarListItemText,
 } from '@/components/sidebar/SidebarList'
-import { loadBeamer } from '@/services/beamer'
-import { useAppSelector } from '@/store'
-import { CookieAndTermType, hasConsentFor } from '@/store/cookiesAndTermsSlice'
-//import { openCookieBanner } from '@/store/popupSlice'
-//import BeamerIcon from '@/public/images/sidebar/whats-new.svg'
+import { BEAMER_SELECTOR } from '@/services/beamer'
 import HelpCenterIcon from '@/public/images/sidebar/help-center.svg'
 import { Divider, Link, ListItem, SvgIcon, Typography } from '@mui/material'
 import DebugToggle from '../DebugToggle'
-import { HELP_CENTER_URL, IS_PRODUCTION } from '@/config/constants'
-import { useCurrentChain } from '@/hooks/useChains'
-import Track from '@/components/common/Track'
-import { OVERVIEW_EVENTS } from '@/services/analytics'
+import { HELP_CENTER_URL, IS_PRODUCTION, NEW_SUGGESTION_FORM } from '@/config/constants'
 import darkPalette from '@/components/theme/darkPalette'
+import SuggestionIcon from '@/public/images/sidebar/lightbulb_icon.svg'
 import ProtofireLogo from '@/public/images/protofire-logo.svg'
-// import IndexingStatus from '@/components/sidebar/IndexingStatus'
 
 const SidebarFooter = (): ReactElement => {
-  //const dispatch = useAppDispatch()
-  const chain = useCurrentChain()
-  const hasBeamerConsent = useAppSelector((state) => hasConsentFor(state, CookieAndTermType.UPDATES))
-
-  useEffect(() => {
-    // Initialise Beamer when consent was previously given
-    if (hasBeamerConsent && chain?.shortName) {
-      loadBeamer(chain.shortName)
-    }
-  }, [hasBeamerConsent, chain?.shortName])
-
-  // const handleBeamer = () => {
-  //   if (!hasBeamerConsent) {
-  //     dispatch(openCookieBanner({ warningKey: CookieAndTermType.UPDATES }))
-  //   }
-  // }
-
```

Comment:
@ivanpochta It is always better to comment out smth, rather than delete. it will be easier to solve merge conflicts in the future roll outs.